daemon/src/wire.rs

```diff
@@ -174,6 +175,10 @@ pub enum MakerToTaker {
         oracle_event_id: BitMexPriceEventId,
         tx_fee_rate: TxFeeRate,
         funding_rate: FundingRate,
+        /// Fee to be paid for the rollover.
+        ///
+        /// The maker comes up with this amount so that both parties are on the same page
+        complete_fee: FeeFlow,
```

Comment:
❓ How does the application react if a different taker version tries to deserialize `ConfirmRollover` with this additional field? Does that work without error or are we breaking compatibility here?

Reply:
I think the cleaner way forward would be to introduce a new version of the `ConfirmRollover` message and distinguish how it will be handled in taker and maker.

Reply:
> ❓ How does the application react if a different taker version tries to deserialize ConfirmRollover with this additional field? Does that work without error or are we breaking compatibility here?

Yes, serde ignores additional fields in the json.

Reply:
> I think the cleaner way forward would be to introduce a new version of the ConfirmRollover message and distinguish how it will be handled in taker and maker.

I don't think this is necessary and unnecessarily adds complecity to our code base.